src/components/SubscriberListSelector/SubscriberListSelector.js

```diff
@@ -7,13 +7,19 @@ import { SubscriberListState } from '../../services/shopify-client';
 import InfiniteScroll from 'react-infinite-scroll-component';
 import { useInfinitePaged } from '../../hooks/useInfinitePaged';
 import { IconMessage } from '../form-helpers/form-helpers';
+import { FormattedMessageMarkdown } from '../../i18n/FormattedMessageMarkdown';
 
-const ListRow = ({ list, selected }) => {
+const ListRow = ({ list, selected, onSelectChange, disabled }) => {
   return (
     <tr>
       <td aria-label="List Check">
-        <label className="dp-list-check">
-          <input type="checkbox" defaultChecked={!!selected} />
+        <label className={'dp-list-check' + (disabled ? ' dp-inputcheck-disabled' : '')}>
+          <input
+            type="checkbox"
+            defaultChecked={!!selected}
```

Comment:
Why do you convert to boolean? The includes method return a boolean. There's no need

Reply:
Good catch!